deploy/crds/operator.ibm.com_v1alpha1_metaoperatorconfig_cr.yaml

```diff
@@ -4,6 +4,16 @@ metadata:
   name: common-service
 spec:
   services:
+  - name: ibm-metering-operator
+    spec:
+      Metering:
+        iamNamespace: kube-system
+        mongodb:
+          host: mongodb.kube-system.svc.cluster.local
+      MeteringUI:
+        iamNamespace: kube-system
+        mongodb:
+          host: mongodb.kube-system.svc.cluster.local
```

Comment:
@horis233 @DanielXLee can we set the spec to {}?

All the default values are set inside the CSV file, this can decouple the service operator and meta-operator.

The service operator controls everything and they don't need to notify meta-operator about its change.

```
spec:
  metering: {}
  meteringUI: {}
```

Reply:
@chenzhiwei yes, we support to set the value to {}. Then meta operator will just use the alm-example to generate crd.

Reply:
done